app/controllers/discover.js

```diff
@@ -76,15 +77,23 @@ export default Ember.Controller.extend({
         }).then(function(results) {
             var hits = results.aggregations.sources.buckets;
             var providers = [];
+            var whiteList = _this.get('whiteListedProviders');
             hits.map(function(each) {
```

Comment:
Is map the right thing to use here? how about ```.filter``` to filter out the results list? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter